pystan/model.py

```diff
@@ -226,6 +238,12 @@ def __init__(self, file=None, charset='utf-8', model_name="anon_model",
                  obfuscate_model_name=True, extra_compile_args=None,
                  allow_undefined=False, include_dirs=None, includes=None):
 
+        tbb_dir = os.getenv("STAN_TBB")
```

Comment:
TBB can be shared by many programs, so user might want to use system tbb files and then distribute resources between multiple programs.

That's why I set it so that user can select what tbb folder to use.